go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rook/rook
 
-go 1.22.6
+go 1.22.5
```

Comment:
I see a different version in toolchain

Reply:
toolchain version will be latest z version

Reply:
>remove specific z version in go mod

The point was to remove the Z version, I am not following it 
Removing mine needinfo and can talk separately

Reply:
please check the slack thread for more details